src/sentry/sentry_metrics/visibility/metrics_blocking.py

```diff
@@ -183,14 +189,18 @@ def get_blocked_metrics_for_relay_config(project: Project) -> BlockedMetricsRela
         sentry_sdk.capture_exception(e)
         # In case of a malformed configuration, we will notify Sentry and return no metrics, since it's an unrecoverable
         # situation, unless we want to force overwrite the config.
-        return BlockedMetricsRelayConfig(deniedNames=[])
+        return BlockedMetricsRelayConfig(deniedNames=[], deniedTags=[])
```

Comment:
maybe function name should say that we return a config, not just the metrics that get blocked  
 
also, isn't it more straight forward to call `BlockedMetrics.load_from_project` than `get_blocked_metrics` with a list of one item then discarding the keys?

Reply:
Yep, the naming is confusing and I was trying to improve it, especially since we can also only block tags.

Reply:
I renamed the dataclasses and functions to be more generic.